packages/@office-iss/react-native-win32/src/Libraries/Components/AccessibilityInfo/AccessibilityInfo.win32.js

```diff
@@ -39,147 +65,286 @@ const _subscriptions = new Map();
-    _subscriptions.delete(handler);
+    const deviceEventName = EventNames.get(eventName);
+    return deviceEventName == null
+      ? {remove(): void {}}
+      : RCTDeviceEventEmitter.addListener(deviceEventName, handler);
   },
 
   /**
-   * Set accessibility focus to a react component.
+   * Set accessibility focus to a React component.
    *
    * See https://reactnative.dev/docs/accessibilityinfo.html#setaccessibilityfocus
    */
-  setAccessibilityFocus: function(reactTag: number): void {
+  setAccessibilityFocus(reactTag: number): void {
     legacySendAccessibilityEvent(reactTag, 'focus');
   },
 
   /**
    * Send a named accessibility event to a HostComponent.
    */
-  sendAccessibilityEvent_unstable: function(
+  sendAccessibilityEvent_unstable(
     handle: ElementRef<HostComponent<mixed>>,
     eventType: AccessibilityEventTypes,
   ) {
-    // route through React renderer to distinguish between Fabric and non-Fabric handles
     // iOS only supports 'focus' event types
-    if (eventType === 'focus') {
-      sendAccessibilityEvent(handle, eventType);
-    } else if (eventType === 'click') {
-      // Do nothing!
+    if (Platform.OS === 'ios' && eventType === 'click') {
+      return;
     }
+    // route through React renderer to distinguish between Fabric and non-Fabric handles
+    sendAccessibilityEvent(handle, eventType);
   },
 
   /**
    * Post a string to be announced by the screen reader.
    *
    * See https://reactnative.dev/docs/accessibilityinfo.html#announceforaccessibility
    */
-  announceForAccessibility: function(announcement: string): void {
-    if (NativeAccessibilityInfo) {
-      NativeAccessibilityInfo.announceForAccessibility(announcement);
+  announceForAccessibility(announcement: string): void {
+    if (Platform.OS === 'android' || Platform.OS === 'windows') {
+      NativeAccessibilityInfo?.announceForAccessibility(announcement);
+    } else {
+      NativeAccessibilityManagerIOS?.announceForAccessibility(announcement);
+    }
+  },
+
+  /**
+   * @deprecated Use `remove` on the EventSubscription from `addEventListener`.
+   */
+  removeEventListener<K: $Keys<AccessibilityEventDefinitions>>(
+    eventName: K,
+    handler: (...$ElementType<AccessibilityEventDefinitions, K>) => void,
+  ): void {
+    // NOTE: This will report a deprecation notice via `console.error`.
+    const deviceEventName = EventNames.get(eventName);
+    if (deviceEventName != null) {
+      // $FlowIgnore[incompatible-cast]
+      (RCTDeviceEventEmitter: EventEmitter<$FlowFixMe>).removeListener(
+        'deviceEventName',
+        // $FlowFixMe[invalid-tuple-arity]
+        handler,
+      );
+    }
+  },
+
+  /**
+   * Get the recommended timeout for changes to the UI needed by this user.
+   *
+   * See https://reactnative.dev/docs/accessibilityinfo.html#getrecommendedtimeoutmillis
+   */
+  getRecommendedTimeoutMillis(originalTimeout: number): Promise<number> {
+    if (Platform.OS === 'android') {
```

Comment:
Just a double-check: All the other checks had windows follow android api's except here.
Only pointing out to make sure intentional...

Reply:
If the new native code will never be called from JS, then I'd say don't add it (cpp implementation).